test/functional/mempool_updatefromblock.py

```diff
@@ -18,7 +18,7 @@
 class MempoolUpdateFromBlockTest(BitcoinTestFramework):
     def set_test_params(self):
         self.num_nodes = 1
-        self.extra_args = [['-limitdescendantsize=1000', '-limitancestorsize=1000', '-limitancestorcount=100']]
+        self.extra_args = [['-limitdescendantsize=1000', '-limitancestorsize=1000', '-limitancestorcount=100', "-limitclustersize=1000"]]
```

Comment:
the rest of these args can be deleted and the test passes

Reply:
Thanks, removed.